code/game/objects/items/weapons/melee/transforming.dm

```diff
@@ -23,6 +26,20 @@
 	if(transform_weapon(user))
 		clumsy_transform_effect(user)
 
+/obj/item/weapon/melee/transforming/attack(mob/living/target, mob/living/carbon/human/user)
+	var/nemesis_faction
+	for(var/F in target.faction)
+		if(F in nemesis_factions)
+			nemesis_faction = TRUE
+			if(!bonus_active)
+				force += faction_bonus_force
+				bonus_active = TRUE
+			nemesis_effects(user, target)
+	if(!nemesis_faction && bonus_active)
```

Comment:
Wouldn't that make it so that you could smack a goliath, then keep the extra damage from it until after making an attack against any other mob?